upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -451,6 +451,29 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 		}
 	}
 
+	// @check the node-local-dns has not been disabled
+	NodeLocalDNS := b.cluster.Spec.KubeDNS.NodeLocalDNS
+	if kubeDNS.Provider == "CoreDNS" && NodeLocalDNS.Enabled {
+		{
+			key := "nodelocaldns.addons.k8s.io"
+			version := "1.18.0-alpha.2"
```

Comment:
Do you consider the addon alpha? If not, I would omit the alpha part here.
By the time this is released, I guess the manifest and such is GA from K8s side.

Reply:
It makes sense. the upstream manifest from my understanding reached GA so I think we are fine here to set it stable.